### PR TITLE
Adds a new vending machine that vends fitness related products

### PR DESCRIPTION
**Pull request by Yoshax**
Adds a new vendor.

It vends
- Protein Bars
- Milk
- Chocolate Milk
- Protein Shakes
- Empty protein shake flask things
- Diet Pills

Contraband
 - Steroid Syringes

### CODE REVIEW

File: code/game/objects/items/trash.dm [hunk 1]
```diff
@@ -9,11 +9,15 @@
 
 /obj/item/trash/raisins
 	name = "\improper 4no raisins"
-	icon_state= "4no_raisins"
+	icon_state=  "4no_raisins"
```
Comment (Datraen) [inline code/game/objects/items/trash.dm]: Meant to put the space on the other side?

Comment (Yoshax) [inline code/game/objects/items/trash.dm]: Indeed.

**Comment by Raptor1628** [comment]
I always wanted gains in space.

**Comment by Anewbe** [comment]
No water bottles?

**Comment by Datraen** [comment]
:+1:

**Comment by TableTurning** [comment]
https://youtu.be/ckZlj2p8W9M

**Pull request merged**